src/data/faq.json

```diff
@@ -504,16 +504,17 @@
                         ]
                     },
                     {
-                        "title": "[Google/Android] An “Incompatibility warning” card on my Start Screen is indicating that my device is not compatible with Exposure Notifications at all. What does this mean?",
+                        "title": "[Google/Android] A message “Incompatibility warning” on my Start Screen indicates that my device is not compatible with Exposure Notifications at all. What does this mean?",
                         "anchor": "full_incompat",
                         "active": true,
                         "textblock": [
-                            "This does not usually happen on unmodified end-user devices. Therefore, this section doesn't apply to the vast majority of devices. Most devices are, at most, <a href='#part_incompat' target='_blank' rel='noopener'>partially incompatible</a>.",

```

Comment:
I understand that this is not a great introduction to the entry. However, I was worried that users could scroll down to the next entry by accident (also specifically since it now looks very similar), and falsely think that their device cannot bluetooth at all.

Reply:
My problem with the introduction sentence is that for someone who reads this without knowing about the nature of the incompatibility, it would be even more confusing. Because then it is not really clear which variant (partial or full) actually applies to the user.

I agree that this could be a problem if a user scrolls down by accident or ends up reading the wrong entry when they use the FAQ search, but that's not something that can be easily fixed with more text and links. Making the warning messages distinct would be one way to solve it I guess.